mifosng-provider/src/main/java/org/mifosplatform/portfolio/charge/domain/Charge.java

```diff
@@ -84,6 +84,9 @@
 
     @Column(name = "fee_frequency", nullable = true)
     private Integer feeFrequency;
+    
+    @Column(name = "disbursement_charge_type_enum", nullable = false)
```

Comment:
don't add specific configuration of charge type instead you can declare it as seperate charge type like disbursement